### PR TITLE
fix(Touchpad): Add Touchpad to other events translations.

### PR DESCRIPTION
**Pull request by pastaq**
TODO: 
- Add Touchpad & Touchscreen to mouse translations

### CODE REVIEW

**Comment by philmmanjaro** [comment]
Testsed 2f55df5 combined with OGUI a21df1a7. seems to work so far.

**Comment by philmmanjaro** [comment]
This needs a rebase on top of a2e1af842e4a21968f653bda5b95249d29a59663

File: rootfs/usr/share/inputplumber/schema/composite_device_v1.json [hunk 1]
```diff
@@ -57,6 +57,7 @@
               "deck",
               "ds5",
               "ds5-edge",
+              "touchpad",
```
Comment (ShadowApex) [inline rootfs/usr/share/inputplumber/schema/composite_device_v1.json:60]: We also need to add this to the input profile schema

**Review by ShadowApex — APPROVED** [review]

**Review by ShadowApex — APPROVED** [review]

**Comment by philmmanjaro** [comment]
8e18b7cc7ba9c9614b3f9d251dc1c5dbefc7ba81 compiled fine as backport on 0.33.1.

**Pull request merged**